forge/Cargo.toml

```diff
@@ -11,7 +11,7 @@ foundry-config = { path = "./../config" }
 foundry-evm = { path = "./../evm" }
 
 # TODO: Trim down
-ethers = { workspace = true, features = ["solc-full"] }
+ethers = { workspace = true, features = ["solc-full", "rustls"] }
```

Comment:
this is incredibly weird, for some reason this now pulls in openssl again.. 
see failing deny job

Reply:
I can't repro this locally, no clue

Reply:
see 4702967